cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,24 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  it('The user should be able to log in', () => {
+    cy.visit('https://react-redux.realworld.io')
 
+    cy.contains('a', 'Sign in')
+      .click()
+
+    cy.contains('h1.text-xs-center', 'Sign In')
```

Comment:
don't use contains for assertions, use should() instead, check and fix everywhere